components/layout/display_list/webrender_helpers.rs

```diff
@@ -137,6 +154,8 @@ impl DisplayList {
         let mut builder = DisplayListBuilder::new(webrender_pipeline);
         builder.begin();
 
+        builder.dump_serialized_display_list();
```

Comment:
Should this be gaurded by a check for the value of `dump_display_list` debug option? Looks like setting this causes WebRender to serialize any items subsequently [added to the display list](https://github.com/servo/webrender/blob/3a951cec4bf463da545b5bbaf0b492b54ecdd7b2/webrender_api/src/display_list.rs#L1203). Is that not a performance concern, even if this is just the non-legacy layout?

Reply:
I had left this debugging code in accidentally, but I've pushed a new version of the branch that does what you suggest. This is actually a bit useful.